### ISSUE TITLE
Question about UADA loss function

### ISSUE THREAD

**Issue by fuhaha824** [William-wAng618/roboticAttack#12]
Hi,
I noticed the UADA loss function in the code is different from Equation (4) in the paper. The code adds a + 1/celoss term:

```
celoss = output.loss
MSE_Distance, UAD = self.weighted_loss(...)
MSE_Distance = MSE_Distance + 1/celoss 
MSE_Distance.backward()
```

My question is: **Were the UADA results in the paper's tables produced using this + 1/celoss term?**

Thanks
